meilisearch-release-changelog/v1.11.0.md

```diff
@@ -0,0 +1,177 @@
+# v1.11.0 release changelogs
+
+<!-- The following line should ONLY be put PRE-release changelogs -->
+> [!WARNING]
+> Since this is a release candidate (RC), we do NOT recommend using it in a production environment. Is something not working as expected? We welcome [bug reports](https://github.com/meilisearch/meilisearch/issues/new/choose) and [feedback about new features](https://github.com/meilisearch/product/discussions).
+
+Meilisearch v1.11 introduces several changes about hybrid search: usage improvements to prepare a future stabilization of the feature, but also a huge impact on the performance thanks to the binary quantization. Finally, according to your feedback, we improved the federated search usage.
+
+<!-- The following lines should NOT be put in the PRE-release changelogs -->
+🧰 All official Meilisearch integrations (including SDKs, clients, and other tools) are compatible with this Meilisearch release. Integration deployment happens between 4 to 48 hours after sa new version becomes available.
+
+<!-- The following lines should NOT be put in the PRE-release changelogs -->
+Some SDKs might not include all new features. Consult the project repository for detailed information. Is a feature you need missing from your chosen SDK? Create an issue letting us know you need it, or, for open-source karma points, open a PR implementing it (we'll love you for that ❤️).
+
+# New features and updates 🔥
+
+## Experimental - Hybrid search improvements
+
+For a future stabilization, multiple changes have been applied to hybrid search features.
+
+📖 More details about the usage in this [page](https://www.notion.so/meilisearch/v1-11-AI-search-changes-0e37727193884a70999f254fa953ce6e).
+
+Done by @dureuill in #4906, #4920, #4892, and #4938.
+
+### ⚠️ Breaking changes
+
+* When using the semantic or the hybrid search, `hybrid.embedder` is now a **mandatory** parameter in `GET and POST /indexes/{:indexUid}/search`
+* As a consequence, it is now **mandatory** to pass `hybrid` even for full-vector search (with only `vector` and not `q`)
+* `embedder` is now a **mandatory** parameter in `GET and POST /indexes/{:indexUid}/similar`
+* Ignore non-zero `semanticRatio` when `vector` is passed but not `q`: a semantic search will be performed (unless `hybrid.semanticRatio` is null).
```

Comment:
```suggestion
* Ignore non-zero `semanticRatio` when `vector` is passed but not `q`: a semantic search will be performed.
```

Reading again the sentence I think we can remove the confusing parens as it repeating the `non-zero` part in a different way.